tests/vnc_test.go

```diff
@@ -246,17 +247,20 @@ var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][lev
 		It("should allow creating a VNC screenshot in PNG format", func() {
 			filePath := filepath.Join(GinkgoT().TempDir(), "screenshot.png")
 
-			cmd := clientcmd.NewVirtctlCommand("vnc", "screenshot", "--namespace", vmi.Namespace, "--file", filePath, vmi.Name)
-			Expect(cmd.Execute()).To(Succeed())
+			// Sometimes we can see initially a 640x480 resolution if we connect very early
+			By("gathering screenshots until we are past the first boot screen and see the expected 720x400 resolution")
+			Eventually(func() image.Image {
+				cmd := clientcmd.NewVirtctlCommand("vnc", "screenshot", "--namespace", vmi.Namespace, "--file", filePath, vmi.Name)
```

Comment:
I did not check and I rather ask, will we override the previous file?

Reply:
Yes, `os.WriteFile` truncates the file if necessary first.